src/screens/AlarmsScreen.tsx

```diff
@@ -2,30 +2,47 @@ import * as React from 'react';
 import { View, StyleSheet, ScrollView } from 'react-native';
 import DetailedAlarmMetricDisplay from '../components/DetailedAlarmMetricDisplay';
 import { convertArrayToMatrix } from '../utils/helpers';
-import { alarmsMetrics } from '../../sample-data/data';
-import { Parameter } from '../Interfaces/Parameter';
+// import { alarmsMetrics } from '../../sample-data/data';
+import { Row } from '../components/Globals/Row';
+import initalVentilatorConfiguration from '../constants/InitialVentilatorConfiguration';
+import VentilatorConfiguration from 'src/interfaces/VentilatorConfiguration';
 
 export default function AlarmsScreen() {
-  const metrics = convertArrayToMatrix<Parameter>(alarmsMetrics, 4);
+  const metrics = convertArrayToMatrix<VentilatorConfiguration>(
```

Comment:
So this is basically retrieving all the object of the `ventilatorConfiguration` object which have a `name` property defined?

I think you should move the extraction of that into a separate method which better explains what is going on. Better yet, you can out the parameters first into an array, and then pass that into the `convertArrayToMatrix` - that might be more readable and understandable